vllm/engine/llm_engine.py

```diff
@@ -336,6 +403,8 @@ def _init_tokenizer(self, **tokenizer_init_kwargs):
         self.tokenizer = get_tokenizer_group(
             self.parallel_config.tokenizer_pool_config, **init_kwargs)
 
+        return self.tokenizer
```

Comment:
If returning it I think better not to set it here too. Can change the caller to `self.tokenizer = self._init_tokenizer(...`.